capabilities/docker/docker.go

```diff
@@ -0,0 +1,50 @@
+package docker
```

Comment:
I did get stuck here... names gladly acceped. I originally though "host", but that's already in use and does not adequately allow for future cases like K8s or swarm. I did consider `dockerhost` or `dockerserver`, but both also felt overloaded.

Naming things is hard!